tasks/config.yml

```
@@ -195,7 +195,7 @@
 - name: enable IPv6 in FreeSWITCH service 2/2
   lineinfile:
     path: /opt/freeswitch/etc/freeswitch/autoload_configs/event_socket.conf.xml
-    regexp: '<param name="listen-ip" value="127.0.0.1"/>'
+    regexp: '<param name="listen-ip" value="[0-9\.:]+"/>'
```

Could you maybe add all hexadecimal letters? then this would also math other IPv6-Addresses.

---

```suggestion
    regexp: '<param name="listen-ip" value="[a-fA-F0-9.:]+"/>'
```

and you dont need to escape the dot within the set of character (square brackets: `[]`)

---

Thanks ! I updated my pull request.